app/_src/mesh/vulnerability-patching-process.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Vulnerability Patching Process
+badge: enterprise
+---
+
+{{site.mesh_product_name}} is primarily delivered as [binary files](/mesh/{{page.release}}/install) installable artifacts. Kong also offers Docker images with the artifacts preinstalled as a convenience to customers. At the time of release, all artifacts and images are patched, scanned and are free of publicly-known vulnerabilities. 
+
+## Types of Vulnerabilities
+
+Generally, there may be three types of vulnerabilities:
+* In {{site.mesh_product_name}} code
+* In third-party code that {{site.mesh_product_name}} directly links (such as Envoy, CoreDNS, OPA, etc)
+* In third-party code that is part of the convenience Docker image (such as Python, Perl, cURL, etc). This code is not part of {{site.mesh_product_name}}.
+
+Vulnerabilities reported in {{site.mesh_product_name}} code will be assessed by Kong and if the vulnerability is validated, a CVSS3.0 score will be assigned. Based on the CVSS score, Kong will aim to produce patches for all applicable {{site.mesh_product_name}} versions currently under support within the SLAs below. The SLA clock starts from the day the CVSS score is assigned.
+
+For a CVSS 3.0 Critical vulnerability (CVSS > 9.0), Kong will provide a workaround/recommendation as soon as possible.  This will take the shape of a configuration change recommendation, if available. If there is no workaround/recommendation readily available, Kong will use continuous efforts to develop one.  For a CVSS <9.0, Kong will use commercially-reasonable efforts to provide a workaround or patch within the applicable SLA period.
```

Comment:
```suggestion
For a CVSS 3.0 Critical vulnerability (CVSS > 9.0), Kong will provide a workaround or recommendation as soon as possible. This will take the shape of a configuration change recommendation, if available. If there is no workaround or recommendation readily available, Kong will use continuous efforts to develop one. 

For a CVSS < 9.0, Kong will use commercially-reasonable efforts to provide a workaround or patch within the applicable SLA period.
```